src/GearPlanner/useIsLocalEnvironment.ts

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from "react";
+
+const localHostnames = [
+    'localhost',
+    '127.0.0.1',
+    '::1',
+]
+
+export default () => useMemo(() => {
+    return localHostnames.includes(location.hostname)
```

Comment:
Rely on `process.env.NODE_ENV !== "production"`, instead. No need to check the location at all.

Reply:
I changed it accordingly, wasn't sure whether that would be available in a build. But I guess webpack makes it available.